src/components/CardGrid.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { 
+  Card,
+  Grid,
+  Typography,
+  CardContent,
+} from '@mui/material';
+
+const CardGrid = () => (
+  <Card
+    variant="outlined"
+    style={{ margin: 'auto', flexDirection: 'column', maxWidth: '95vw' }}
+  >
+    <Grid
+      container
+      display="flex"
+      direction="row"
+      alignItems="center"
+      justifyContent="center"
+      rowSpacing={2}
+      columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+      padding="20px"
+      margin="5px"
+    >
+      <Grid item md={3}>
```

Comment:
Missing breakpoint for xs screens (mobile).

Should we have one card per row on mobile or two?

(``xs={6}`` or ``xs={12}``)

Reply:
one card per row on mobile